src/shadows-and-depth.md

```diff
@@ -0,0 +1,106 @@
+:::
+
+### 2. Shadow variables for surfaces
+In a digital user interface, there are many floating UI elements like cards, modals, menus, or popovers that can benefit from a shadow effect. Here is a list of different CSS variables that Lime Elements offers:
+
+<limel-example-surface-shadows></limel-example-surface-shadows>
+
+### 3. Shadow variables for highlighting different states
+<limel-example-surface-shadows-states></limel-example-surface-shadows-states>
+
+:::tip
+To give a color around the focused element, the `-focused` variables will use `var(--lime-primary-color)` if it is defined. When it is not defined, they will default to `var(--color-teal-default)`.
+:::
+
+*****
+
+## When not to use shadow in UI design
+Sometimes we may only be trying to use shadows to create a visually distinct section, to separate a part of the interface from other parts.
+
+In theory, having visually distinct areas in the UI increases the ease and speed of finding components or information when users are quickly scanning pages. But instead of using shadows, this visual distinction is better to be done through other ways, such as simply using outlines, dividers, larger headings, or different background colors.
+
+:::warning
+We want to reserve the visual power of shadows to indicate possibility of interaction, to signify a difference in hierarchy, or to indicate temporariness of an element in the UI.
+
+Therefore, we strongly discourage the use of shadows when the only purpose is to create a visually distinct section in the UI.
+
+If this is your only intention, we recommend using a different background color, use different font characteristics, or rely on a good layout design that uses headings, paddings and margins well.
+:::
+<limel-example-shadows-bad-usage></limel-example-shadows-bad-usage>
+
+Sometimes, using the recommended techniques above does not suffice, when you want to visually emphasize on a section of the UI and make it pop out. For such cases, we offer other variables which can be used as `box-shadow` on the element.
+
+<limel-example-surface-shadows-inflated></limel-example-surface-shadows-inflated>
+
+:::tip
+The best visual expression with these variables is achieved when:
+- the element has a sizable `border-radius`
+- there is enough of empty space around the element
+- the element has a light grey as `background-color`, (preferably `var(--contrast-200)`, `var(--contrast-300)` or `var(--contrast-400)`)
+- `background-color` of the surface below the element is not too dark or too light (preferably `var(--contrast-400)`, `var(--contrast-500)` or `var(--contrast-600)`)
+- the element's `background-color` is lighter than the surface below it
+:::
+
+:::warning
+- Do not over use this visual effect on the same screen.
+- The element using this effect should not be too small. Because the effect may make it look like a clickable button.
```

Comment:
```suggestion
- The element using this effect should not be too small, or the effect might make it look like a clickable button.
```

I also don't know if "clickable" needs to be there. Buttons are clickable, that's their main feature.

So perhaps this, if you like it:

```suggestion
- The element using this effect should not be too small, or the effect might make it look like a button.
```